modules/tinymce/src/core/main/ts/api/textpatterns/TextPatterns.ts

```diff
@@ -1,7 +1,14 @@
-import { RawPattern, RawDynamicPatternsLookup, DynamicPatternContext } from '../../textpatterns/core/PatternTypes';
+import { RawPattern, RawDynamicPatternsLookup, DynamicPatternContext, DynamicPatternsLookup, Pattern, BlockCmdPattern, BlockFormatPattern, InlineCmdPattern, InlinePattern, InlineFormatPattern } from '../../textpatterns/core/PatternTypes';
 
 export {
+  Pattern,
   RawPattern,
+  DynamicPatternsLookup,
   RawDynamicPatternsLookup,
-  DynamicPatternContext
+  DynamicPatternContext,
+  BlockCmdPattern,
```

Comment:
Should `BlockPattern` also be exposed here given we're exposing the `InlinePattern` type which is equivalent.

Reply:
Yeah. I think so for consistency. I'll update.

Reply:
Exposed BlockPattern and tested it was exposed.